yarn-error.log

```diff
@@ -1,119 +0,0 @@
-Arguments: 
```

Comment:
You should have this file in .gitignore
I recommend to use this one - https://github.com/github/gitignore/blob/master/Node.gitignore

Reply:
Yeah good call

Reply:
ok, added this one